protos/ensembl/production/metadata/ensembl_metadata.proto

```diff
@@ -143,6 +149,14 @@ message Release {
   string site_uri = 7;
 }
 
+message Organism {
+  string display_name = 1;
```

Comment:
I believe Organism has a field we would like to retrieve, `scientific_parlance_name` (or is it merge with display name?)